lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu
+
+  attr_reader :items
+
+  def initialize
+    @items = [{ name: "Haggis Supper", price: "10.99" },
```

Comment:
As this is in initialize if a user updates the menu this will be overridden next time the menu is created. Maybe consider having an empty array of hashes.